src/index.html

```diff
@@ -11,7 +11,166 @@
+        <label class="form__field">
+          Full date of birth:
+          <input 
+          class="form__input" 
+          type="date" 
+          name="user-birth" 
+          required
+          >
+        </label>
+        <label class="form__field">
+          I accept the term of the agreement
+          <input 
+           class="form__input" 
+           type="checkbox"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend class="form__title">Registration</legend>
+        <label class="form__field">
+          E-mail:
+          <input
+            class="form__input"
+            type="email"
+            name="user-email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label class="form__field">
+          Password:
+          <input
+            class="form__input"
+            type="password"
+            name="user-password"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend class="form__title">An interesting fact about you!</legend>
+        <label class="form__field">
+          Do you like cats? Yes
+          <input
+            class="form__input"
+            type="radio"
+            name="user-love-cats"
+            value="yes"
+            required
+          >
+          No
+          <input
+            class="form__input"
+            type="radio"
+            name="user-love-cats"
+            value="no"
+            required
+          >
+        </label>
+        <label class="form__field">
+          What is your favorite color?
+          <input 
+            class="form__input" 
+            type="color" 
+            name="user-color"
+          >
+        </label>
+        <label class="form__field">
+          What time do you go to bed?
+          <input
+            class="form__input"
+            type="time"
+            name="user-go-bed-time"
+          >
+        </label>
+        <label class="form__field">
+          What are your favorite brand of cars?
+          <select 
+            name="user-cars" 
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+        <label class="form__field">
+          How do you rate our work?
+          <input
+            class="form__input"
+            type="range"
+            name="user-rate"
+            value="0"
+            min="0"
+            max="100"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend class="form__title">Additional info:</legend>
+
+        <label class="form__field">
+          Comments:
+          <textarea 
+          name="user-comments" 
+          rows="2" 
+          cols="15"
+          >
+          </textarea>
+        </label>
+
```

Comment:
```suggestion
          Comments:
          <textarea 
            name="user-comments" 
            rows="2" 
            cols="15"
          ></textarea>
        </label>

```

давай виправимо поведінку textarea

https://user-images.githubusercontent.com/102413088/190385178-12a97ca4-f4dd-44c9-9f3c-bee259bd4f2c.mov

https://mate.academy/learn/css-advanced/forms#/video/885